frontend/public/.eslintrc

```diff
@@ -8,7 +8,8 @@
   "rules": {
     "no-unused-vars": [
       "error", {
-        "argsIgnorePattern": "^__"
+        "argsIgnorePattern": "^__",
+        "ignoreRestSiblings": true
```

Comment:
This means that in `const { a, ...others } = thing`, `a` won't be treated as unused, even if it is never used again. This allows object spread `...` to be used for omitting properties from an object without triggering errors. https://eslint.org/docs/latest/rules/no-unused-vars#ignorerestsiblings